values/registry1-values.yaml

```diff
@@ -5,6 +5,36 @@ configConnectionPooler:
   connection_pooler_image: "registry1.dso.mil/ironbank/opensource/zalando/pgbouncer:1.21.0"
 configLogicalBackup:
   logical_backup_docker_image: "registry1.dso.mil/ironbank/opensource/zalando/logical-backup:v1.8.2"
-# Note there is not a spilo image in registry1
+# Note there is not a spilo or postgres-exporter image in registry1
 configGeneral:
   docker_image: "ghcr.io/zalando/spilo-15:3.2-p1"
+  sidecars:
+    - name: "exporter"
+      image: "quay.io/prometheuscommunity/postgres-exporter:v0.15.0"
```

Comment:
this should be the registry1 image such as 
registry1.dso.mil/ironbank/opensource/prometheus/postgres-exporter:v0.15.0